db/schema.rb

```diff
@@ -23,18 +23,60 @@
     t.datetime "image_updated_at"
   end
 
+  create_table "admin_admission_eligibilities", force: :cascade do |t|
+    t.string   "name"
+    t.text     "description"
+    t.datetime "created_at",  null: false
+    t.datetime "updated_at",  null: false
+  end
+
   create_table "admin_departments", force: :cascade do |t|
     t.string   "name"
     t.string   "short_form"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
   end
 
+  create_table "admin_notice_boards", force: :cascade do |t|
```

Comment:
Why it is render ? Is this story told you to make `admin_notice_boards` table ? Please remove this un necessary table.